src/processTargets/modifiers/scopeTypeStages/RegexStage.ts

```diff
@@ -38,7 +38,10 @@ class RegexStage implements ModifierStage {
     for (let i = start.line; i <= end.line; ++i) {
       this.getMatchesForLine(editor, i).forEach((range) => {
         // Regex match and selection intersects
-        if (range.end.isAfterOrEqual(start) && range.end.isBeforeOrEqual(end)) {
+        if (
+          range.end.isAfterOrEqual(start) &&
+          range.start.isBeforeOrEqual(end)
```

Comment:
I don't mind recording them but can you give some examples of what would test this? I don't really remember what the failing case was since I only saw it in that one discord call.

Reply:
cc/ @AndreasArvidsson

Reply:
`aa|aa bb|bb`
`"clear every paint"`